draft-ietf-quic-http.md

```diff
@@ -1651,6 +1670,18 @@ The entries in the following table are registered by this document.
 | Push Stream      | 0x50   | {{server-push}}            | Server |
 | ---------------- | ------ | -------------------------- | ------ |
 
+Additionally, each code of the format `0x1f * N` for values of N in the
+range (0..8) (that is, `0x00`, `0x1f`, etc., through `0xf8`), the following
```

Comment:
Let's list them all; it's only 8 values. ('0x00', '0x1f', '0x3e', '0x5d', '0x7c', '0x9b', '0xba', '0xd9', '0xf8').